demo_outerframes.py

```diff
@@ -0,0 +1,60 @@
+import traceback
```

Comment:
Did you mean to add this file?

Reply:
yeah, allows uncommenting line 15 to compare with the default traceback. I'll probably remove all the commented stuff before merging this